### PR TITLE
Just to make sure #824 Doesn't occur...

### PR DESCRIPTION
**Pull request by CortexPE**
### Purpose and Effect of This Pull Request
Added an if statement Just to make sure #824 Doesn't occur...
It works for me w/o problems...

### Testing
This is an inconsistent bug. We need @KenMizz to test it for us.

### CODE REVIEW

File: src/pocketmine/Server.php [hunk 1]
```diff
@@ -336,7 +336,7 @@ private function get_calling_class() : string {
 		for ( $i=1; $i<count( $trace ); $i++ ) {
 			if ( isset( $trace[$i] ) )
 				if ( $class != $trace[$i]['class'] )
-					return $trace[$i]['class'];
+					if(!is_null($trace[$i]['class'])) return $trace[$i]['class'];
 		}
 		return "null";
```
Comment (NidPlays) [inline src/pocketmine/Server.php]: problem might be here @CortexPE

**Comment by VCraftMCPE** [comment]
https://github.com/64FF00/stop-making-pmmp-forks/issues/12

**Comment by CortexPE** [comment]
@KenMizz Please test it again 😄 
(I added another commit... That's why.)

File: src/pocketmine/Server.php (hunk 1 above)
Comment (CortexPE) [inline src/pocketmine/Server.php]: it is.

**Pull request closed without merging**